infrastructure/arm_templates/cosmosdb/errors.json

```diff
@@ -0,0 +1,31 @@
+{
+    "code": "ExportTemplateCompletedWithErrors",
```

Comment:
Skip over the json files.

This one is a bit different in that it is the warnings about what couldn't be exported. However, none of these resources are in the CLI scripts, so I think we're ok!